github-actions/verify-pr-title/package.json

```diff
@@ -17,13 +17,14 @@
 	},
```

Comment:
While you are in here, do you want to update our dependencies for this github action to the latest for @actions/core and @actions/github?

Reply:
Yeah, no reason not to, I think.

Reply:
Can easily update `@actions/core`. Updating `@actions/github` causes a compile failure that I'm looking into.